tox.ini

```diff
@@ -24,9 +22,7 @@ commands =
 
 [testenv:benchmark]
 skip_install = True
-deps =
-    asv
-    virtualenv
```

Comment:
You need to add `virtualenv` to the "benchmarks" extra if you're removing it here; `asv` needs it installed in order to create its environments.  (I also don't believe there's a reason to include `morecontext` in the extra, so that should be removed as well.)